src/modules/ce/components/referral/ReferralStepDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box } from '@mui/material';
+import React, { useMemo } from 'react';
+import {
+  AssigneesIcon,
+  StepCalendarIcon,
+} from '@/components/elements/SemanticIcons';
+import ReferralStepDatum from '@/modules/ce/components/referral/ReferralStepDatum';
+import {
+  formatRelativeDate,
+  parseHmisDateString,
+  stringifyArray,
+} from '@/modules/hmis/hmisUtil';
+import {
+  CeReferralStepStatus,
+  CeReferralStepSummaryFieldsFragment,
+} from '@/types/gqlTypes';
+
+const ReferralStepDetails: React.FC<{
+  step: CeReferralStepSummaryFieldsFragment;
+}> = ({ step }) => {
+  const { status, availableAt, updatedAt, assignees } = step;
+
+  const dateText = useMemo(() => {
+    if (status === CeReferralStepStatus.Unavailable) return;
+    if (status === CeReferralStepStatus.Completed) {
+      const date = parseHmisDateString(updatedAt);
+      if (date) return `Completed ${formatRelativeDate(date)}`;
+    }
+    const date = parseHmisDateString(availableAt);
+    if (date) return `Available since ${formatRelativeDate(date)}`;
```

Comment:
```suggestion
    if (date) return `Available ${formatRelativeDate(date)}`;
```
Thoughts on that? "Available Today", "Available 4 days ago" etc..

Something about "available since 3 days ago" feels overly verbose to me

Reply:
yep agreed 100%!